app/backend/src/couchers/models.py

```diff
@@ -768,10 +770,10 @@ class Page(Base):
 
     __tablename__ = "pages"
 
-    id = Column(BigInteger, primary_key=True)
+    id = Column(BigInteger, communities_seq, primary_key=True)
 
     type = Column(Enum(PageType), nullable=False)
-    thread_id = Column(ForeignKey("threads.id"), nullable=False, unique=True, index=True)
+    thread_id = Column(ForeignKey("threads.id"), nullable=True, unique=True, index=True)
```

Comment:
Will this go back to False once threads are in? Or do pages not necessarily have a conversation?

Reply:
My thinking here was that we don't need the empty thread object there until someone adds a comment, at which point we can dynamically create the thread, hence null initially.